### PR TITLE
User.authorizations() doesn't currently keep track of the AuthorizationProvider that contributed authorizations

### PR DESCRIPTION
**Pull request by stephanebastian**
@pmlopes Could you please review ?

### CODE REVIEW

File: vertx-auth-common/src/main/java/io/vertx/ext/auth/Authorizations.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package io.vertx.ext.auth;
```
Comment (pmlopes) [inline vertx-auth-common/src/main/java/io/vertx/ext/auth/Authorizations.java:13]: header missing

File: vertx-auth-common/src/main/java/io/vertx/ext/auth/impl/AuthorizationsImpl.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
+package io.vertx.ext.auth.impl;
```
Comment (pmlopes) [inline vertx-auth-common/src/main/java/io/vertx/ext/auth/impl/AuthorizationsImpl.java:13]: header missing

**Review by pmlopes — COMMENTED** [review]
Just missing the copyright headers, other than that I'll just merge it ;-)

File: vertx-auth-common/src/main/java/io/vertx/ext/auth/Authorizations.java (hunk 1 above)
Comment (stephanebastian) [reply vertx-auth-common/src/main/java/io/vertx/ext/auth/Authorizations.java:13]: oups... fixed ;)

**Review by pmlopes — APPROVED** [review]

**Pull request merged**